server/mergin/app.py

```diff
@@ -177,7 +177,6 @@ def create_app(public_keys: List[str] = None) -> Flask:
         arguments={"title": "Mergin"},
         options={"swagger_ui": Configuration.SWAGGER_UI},
         validate_responses=True,
-        pythonic_params=True,
```

Comment:
I found it used only it this new v2 blueprint, so in functions params you pass `id_` instead of `id` but found it rather confusing. If we want it this way, then we should propagate it everywhere.